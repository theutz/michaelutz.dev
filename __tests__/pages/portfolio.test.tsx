import { render, screen } from '@testing-library/react'
import Portfolio from 'pages/portfolio'

describe('<Portfolio />', () => {
  it('renders', () => {
    // Arrange

    // Act
    const { asFragment } = render(<Portfolio />)

    // Assert
    expect(asFragment()).toMatchSnapshot()
  })
})
