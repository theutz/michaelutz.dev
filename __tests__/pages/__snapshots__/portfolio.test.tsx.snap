// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Portfolio /> renders 1`] = `
<DocumentFragment>
  <div
    class="relative py-16 bg-white"
    data-cy="page-portfolio"
    data-testid="under-construction"
  >
    <div
      aria-hidden="true"
      class="hidden absolute top-0 inset-x-0 h-1/2 bg-gray-50 lg:block"
    />
    <div
      class="mx-auto bg-indigo-600 max-w-7xl lg:bg-transparent lg:px-8"
    >
      <div
        class="lg:grid lg:grid-cols-12"
      >
        <div
          class="relative z-10 lg:col-start-1 lg:row-start-1 lg:col-span-4 lg:py-16 lg:bg-transparent"
        >
          <div
            aria-hidden="true"
            class="absolute inset-x-0 h-1/2 bg-gray-50 lg:hidden"
          />
          <div
            class="max-w-md px-4 mx-auto sm:max-w-3xl sm:px-6 lg:max-w-none lg:p-0"
          >
            <div
              class="aspect-w-10 aspect-h-6 sm:aspect-w-2 sm:aspect-h-1 lg:aspect-w-1"
            >
              <div
                style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
              >
                <noscript />
                <img
                  alt=""
                  class="shadow-2xl rounded-3xl"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover; object-position: center;"
                />
              </div>
            </div>
          </div>
        </div>
        <div
          class="relative bg-indigo-600 lg:col-start-3 lg:row-start-1 lg:col-span-10 lg:rounded-3xl lg:grid lg:grid-cols-10 lg:items-center"
        >
          <div
            aria-hidden="true"
            class="absolute inset-0 hidden overflow-hidden rounded-3xl lg:block"
          >
            <svg
              aria-hidden="true"
              class="absolute bottom-full left-full transform translate-y-1/3 -translate-x-2/3 xl:bottom-auto xl:top-0 xl:translate-y-0"
              fill="none"
              height="384"
              viewBox="0 0 404 384"
              width="404"
            >
              <defs>
                <pattern
                  height="20"
                  id="64e643ad-2176-4f86-b3d7-f2c5da3b6a6d"
                  patternUnits="userSpaceOnUse"
                  width="20"
                  x="0"
                  y="0"
                >
                  <rect
                    class="text-indigo-500"
                    fill="currentColor"
                    height="4"
                    width="4"
                    x="0"
                    y="0"
                  />
                </pattern>
              </defs>
              <rect
                fill="url(#64e643ad-2176-4f86-b3d7-f2c5da3b6a6d)"
                height="384"
                width="404"
              />
            </svg>
            <svg
              aria-hidden="true"
              class="absolute top-full transform -translate-y-1/3 -translate-x-1/3 xl:-translate-y-1/2"
              fill="none"
              height="384"
              viewBox="0 0 404 384"
              width="404"
            >
              <defs>
                <pattern
                  height="20"
                  id="64e643ad-2176-4f86-b3d7-f2c5da3b6a6d"
                  patternUnits="userSpaceOnUse"
                  width="20"
                  x="0"
                  y="0"
                >
                  <rect
                    class="text-indigo-500"
                    fill="currentColor"
                    height="4"
                    width="4"
                    x="0"
                    y="0"
                  />
                </pattern>
              </defs>
              <rect
                fill="url(#64e643ad-2176-4f86-b3d7-f2c5da3b6a6d)"
                height="384"
                width="404"
              />
            </svg>
          </div>
          <div
            class="relative max-w-md px-4 py-12 mx-auto space-y-6 sm:max-w-3xl sm:py-16 sm:px-6 lg:max-w-none lg:p-0 lg:col-start-4 lg:col-span-6"
          >
            <h2
              class="text-3xl font-extrabold text-white"
              id="join-heading"
            >
              Under Construction
            </h2>
            <p
              class="text-lg text-white"
            >
              Right now, this website is my best project for you to checkout what I've been up to.
            </p>
            <a
              class="block w-full px-5 py-3 text-base font-medium text-center text-indigo-700 bg-white border border-transparent shadow-md rounded-md hover:bg-gray-50 sm:inline-block sm:w-auto"
              href="/"
            >
              Return to Home
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
